packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -249,6 +254,10 @@ declare module '@docusaurus/plugin-content-blog' {
      * Used to generate the page h1 heading, tab title, and pagination title.
      */
     readonly title: string;
+    /**
+     * Used to generate the <head><title> and <meta>.
+     */
+    readonly title_meta?: string;
```

Comment:
The solution we implement here is temporary until I find a better way (where `title` will be used in priority for SEO)

We don't need to add this new `metadata.title_meta` because we'll likely remove it in the future. `frontMatter.title_meta` will be enough for now.